### DISCUSSION TITLE
How to proceed for the removal of the old compat table macros?

### DISCUSSION THREAD

**Discussion by teoli2003** [mdn/mdn-community#73] [Localization]
Hi, @mdn/localization-team-leads!

Last year, on mdn/content, we removed the old compatibility tables and replaced the mess of HTML tables and odd macros with a single macro, reading the data from mdn/browser-compat-data, with a key set in the Yaml header of the page.

We have now in mdn/yari a bunch of macros, no more used at all on mdn/content but that have more than 1000 occurrences in mdn/translated-content:
![image](https://user-images.githubusercontent.com/1466293/167854510-ca8f0b9c-1f39-4c54-b75b-9d27748c3af4.png)

(and `{{property_prefix}}` too)

### How to remove these efficiently?

My 2 🪙 :

Maybe we should remove the _Browser compatibility_ and _Specifications_ sections of all translated pages, add the Yaml `spec_url` key in mdn/content and replace them with the two macros, like in mdn/content.

I think this would improve a lot the quality of the translated pages while helping get closer to actually deleting these macros.

This is just an idea, a conversation starter, so feel free to comment!

**Comment by cw118** [comment]
(pinging @mdn/localization-team-leads)

**Comment by SphinxKnight** [comment]
Cross-linking to https://github.com/mdn/translated-content/issues/5618
